repo/packages/K/kubernetes/1/config.json

```diff
@@ -81,6 +81,15 @@
           "minimum": 0,
           "type": "integer"
         },
+        "resource-roles": {
```

Comment:
I kind of expected that only the apiserver would need to be public once they were split up. Tho once component-registration goes through we might have an explosion of components that may host their own APIs. I'm not sure if they're gonna all proxy through the central apiserver or not.

But for now i'd assume apiserver-proxy is the only one that need to be public (which is not the same as kubectl proxy, fyi).